src/corelibs/U2Gui/src/ObjectViewModel.cpp

```diff
@@ -221,6 +223,9 @@ void GObjectView::sl_onDocumentRemoved(Document *d) {
     }
 }
 
+void GObjectView::sl_onDocumentLoadedStateChanged() {
```

Comment:
Make this function pure virtual

Reply:
This is not correct: this method is not required to be implemented if nothing should be done on document state change (implementation is optional)